tests/unit/job/test_query.py

```diff
@@ -1316,37 +1343,6 @@ def test_result_w_empty_schema(self):
         self.assertEqual(result.location, "asia-northeast1")
         self.assertEqual(result.query_id, "xyz-abc")
 
-    def test_result_invokes_begins(self):
```

Comment:
As of https://github.com/googleapis/python-bigquery/pull/967 released in google-cloud-bigquery 3.0.0, the _begin method is no longer used for query jobs.